test/libsolidity/smtCheckerTests/operators/compound_bitwise_and_uint.sol

```diff
@@ -0,0 +1,34 @@
+pragma experimental SMTChecker;
+
+contract C {
+	function f(bool b) public pure {
+		uint v = 1;
+		if (b)
+			v &= 1;
+		assert(v == 1);
+
+		v = b ? 7 : 3;
+		v &= (b ? 3 : 7);
+		assert(v != 3); // fails, as 7 & 3 = 3
+
+		uint c = 0;
+		c &= v;
+		assert(c == 0);
+
+		uint8 x = 0xff;
+        uint16 y = 0xffff;
+        y &= x;
+        assert(y == 0xff);
+        assert(y == 0xffff);
```

Comment:
I guess this one fails?

Reply:
Yes, should have added comment as well. Adding one now :)